ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -76,22 +103,47 @@ const DuccTextInput = (props) => {
   );
 };
 
-const InitialsAgreement = (props) => {
-  return (
-    <div style={{ display: 'flex', marginTop: '0.5rem' }}>
+const SignedText = (props: { text: string }) => (
+  // add 2 spaces before and after to simulate a signature line
+  <span style={styles.signedText}>&nbsp;&nbsp;{props.text}&nbsp;&nbsp;</span>
+);
+
+interface ReadOnlyProps extends DuccTextInputProps {
```

Comment:
What does `extends DuccTextInputProps` do here?

Reply:
it means that it includes all props in `DuccTextInputProps` in addition to those it adds here

Reply:
Oh, my mistake. I did not notice that props was being passed to DUCCTextInput. I thought that only signatureState was being used.